scripts/globals/items.lua

```diff
@@ -290,6 +315,7 @@ xi.items =
     KODACHI_OF_TRIALS               = 17773,
     MUMEITO                         = 17809,
     TACHI_OF_TRIALS                 = 17815,
+    ANIMATOR_P1                     = 17857,
```

Comment:
I like _P1, _P2 as a pattern

Reply:
P for "Plus" works as long as we don't hit anything that actually has that as part of its text string. If that happens someday we can just put the whole word.

Reply:
we may eventually need "minus" representation also. Some items are named that way but atm none would need handled here.

Reply:
I would go with PLUS_1, mainly becouse it isnt much larger and I dont imediately read it as "Player one" but...